Cheatsheet.md

```diff
@@ -0,0 +1,504 @@
+# Cheatsheet
+## Commonly used functions on [Array](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Arrays)
+### [1. map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)
+
+**Description:**
+Creates an array by calling a specific function on each element present in the parent array.
+
+**Syntax:**
+* _array.map(function(currentValue, index, arr), thisValue)_

```

Comment:
Only include the first parameter, currentValue instead of the others which are rarely used.

The same goes for all callbacks in this doc. Only include the first argument.

Reply:
Sir, are you referring to parameters of _function_ or the parameters of _array.map( )_?